locales/ar/common.json

```diff
@@ -1,6 +1,9 @@
 {
   "location": "الموقع",
   "date": "تاريخ",
+  "extras": "إضافات",
+  "categories": "الفئات",
+  "news": "الأخبار",
```

Comment:
Are those professional translations?

Reply:
Farsi translations are missing :)

Reply:
Yep, These are the constants sheet, Farsi is missing there as well

Reply:
Farsi has been added recently, french aswell :)

Reply:
Farsi  is done, french will be added in WEBAPP-133